allure-report-face/package.json

```diff
@@ -45,6 +45,7 @@
     "proxyquire": "^1.7.3",
     "style-loader": "^0.12.4",
     "underscore.string": "^3.2.2",
+    "url-loader": "^0.5.7",
```

Comment:
We are not using URL loader because CSP policy in Jenkins. It is not allowed to use inline data-URI pictures